schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,222 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Network",
+  "description": "Microsoft Network Resource Types",
+  "resourceDefinitions": {
+    "trafficManagerProfiles": {
+      "type": "object",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2015-11-01"
+          ]
+        },
+        "name": {
```

Comment:
Actually you do not need this, deploymentTemplate.json has defined that and make that "required".